staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4996,11 +4996,21 @@ type EndpointPort struct {
 	// +optional
 	Protocol Protocol `json:"protocol,omitempty" protobuf:"bytes,3,opt,name=protocol,casttype=Protocol"`
 
-	// The application protocol for this port.
+	// Used as a hint for implementations to
```

Comment:
Is there a reason you're line-breaking at 40?  It looks like a haiku. :)

Reply:
lol :) 
probably not, i have opened it a while ago so do not remember why I did it.
Is there a format I should follow though?  80?

Reply:
Yep, 80 is what I've seen most commonly in k8s